::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 11 Jul 2017 18:24:24 +0200
Subject: [PATCH] mcp/config: AArch64: Use UEFI arm64 image

While at it, bump default vCPU number from 2 to 6.

Signed-off-by: Guillermo Herrero <Guillermo.Herrero@enea.com>
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/config/scenario/virtual/defaults.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mcp/config/scenario/virtual/defaults.yaml b/mcp/config/scenario/virtual/defaults.yaml
index b841e88..17fbbfc 100644
--- a/mcp/config/scenario/virtual/defaults.yaml
+++ b/mcp/config/scenario/virtual/defaults.yaml
@@ -1,6 +1,6 @@
-base_image: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
+base_image: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-arm64-uefi1.img
 virtual:
   default:
-    vcpus: 2
+    vcpus: 6
     ram: 4096

